tests/conftest.py

```
@@ -73,8 +92,203 @@ def _override_get_db():
+            # Create a file inventory record to satisfy FK
+            file_inv = file_inventory_factory()
+            file_inventory_id = file_inv.id
+
+        job = RemoteTransferJob(
+            file_inventory_id=file_inventory_id,
+            remote_connection_id=remote_connection.id,
+            remote_monitored_path_id=remote_monitored_path.id,
+            direction=direction,
+            status=status,
+            total_size=file_size,
+            source_path=f"/tmp/{file_name}",
```

Ruff (0.15.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)